google-cloud-bigquery/pom.xml

```diff
@@ -0,0 +1,547 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2022 Cask Data, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.cdap.plugin</groupId>
+    <artifactId>google-cloud</artifactId>
+    <version>0.21.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>io.cdap.plugin</groupId>
+  <artifactId>google-cloud-bigquery</artifactId>
+  <version>0.21.0-SNAPSHOT</version>
+  <name>Google Cloud BigQuery Plugins</name>
+  <description>BigQuery GCP Plugins</description>
+
+  <distributionManagement>
```

Comment:
Do these change for submodules? I think it would keep the submodule pom files simpler if the common parts like distributionManagement, repository, checkstyle, build and even the common dependencies like cdap-api is moved to the top level pom. wdyt?

Reply:
This is a great point. I will consolidate some of the common dependencies in the parent pom (CDAP API classes that are provided and test dependencies)

Reply:
nit: Move the <distributionManagement> section and <repositories> section also to parent pom?